ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -567,3 +567,36 @@ def get_blockpools_compression_status_from_storagesystem(self) -> tuple:
             self.validation_loc["storagecluster-blockpool-details-compress-status"]
         )
         return compression_status_blockpools_tab, compression_status_blockpools_details
+
+    def verify_odf_operator_in_installed_operator(self) -> bool:
+        """
+        Function to validate either ODF operator pod is up and running post ODF installation,
+        expectation is only ODF operator should be running in Installed operators tab other operators shouldn't be there
+        This function is only written for 4.9+ versions
+
+        Returns:
+        True: If only odf operator is present in the UI
+        False: If ocs operator is also present in the UI
+        """
+        if (
+            self.ocp_version_semantic >= version.VERSION_4_9
+            and self.ocs_version_semantic >= version.VERSION_4_9
+        ):
+            logger.info("Navigating to Installed Operator Page")
+            self.navigate_installed_operators_page()
+            logger.info("Searching for Openshift Data Foundation Operator")
+            odf_opeartor_presence = self.wait_until_expected_text_is_found(
+                locator=self.validation_loc["odf-operator"],
+                timeout=1,
+                expected_text="OpenShift Data Foundation",
+            )
```

Comment:
Add log info `logger.info("Searching for Openshift Container Storage Operator")`

Reply:
added this.